package.json

```diff
@@ -197,31 +197,38 @@
         "category": "Axon Ivy"
       }
     ],
-    "configuration": {
-      "title": "Axon Ivy Engine",
-      "properties": {
-        "runEmbeddedEngine": {
-          "type": "boolean",
-          "default": true,
-          "description": "Start embedded Axon Ivy Engine. If set false you have to define host & port where your Engine runs.",
-          "order": 0
-        },
-        "engineUrl": {
-          "type": "string",
-          "default": "http://localhost:8080/",
-          "description": "Url of the Axon Ivy Engine that will be used.",
-          "order": 1
+    "configuration": [
+      {
+        "title": "Axon Ivy",
+        "properties": {
+          "engine.runEmbedded": {
+            "type": "boolean",
+            "default": true,
+            "description": "Start embedded Axon Ivy Engine. If set false you have to define host & port where your Engine runs.",
+            "order": 0
+          },
+          "engine.url": {
+            "type": "string",
+            "default": "http://localhost:8080/",
+            "description": "Url of the Axon Ivy Engine that will be used.",
+            "order": 1
+          },
+          "project.excludePattern": {
+            "type": "string",
+            "default": "",
```

Comment:
as default you could use something like `**/target/` or how would the pattern look like if we do not want to look into target folders?

Reply:
`**/target/**`would it be